src/org/thoughtcrime/securesms/ProfileGalleryFragment.java

```diff
@@ -248,10 +228,20 @@ public boolean onPrepareActionMode(ActionMode mode, Menu menu) {
     @Override
     public boolean onActionItemClicked(ActionMode mode, MenuItem menuItem) {
```

Comment:
Just wondering if this can be put into MessageSelectorFragment too, but you probably can judge this better than me, maybe it doesn't make sense